core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -418,6 +432,15 @@ object GameStarter {
         for (civ in gameInfo.civilizations.filter { !it.isBarbarian() && !it.isSpectator() }) {
             val startingLocation = startingLocations[civ]!!
 
+            //Trigger any global uniques that should triggered.
+            //We may need the starting location for some uniques, which is why we're doing it now
+            for (unique in ruleset.globalUniques.uniqueObjects)
```

Comment:
Unify these into one for loop - for unique in a.asSequence() + b.asSequence()
Since we want to treat them the same way